example/cr-detailed.yaml

```diff
@@ -32,6 +32,12 @@ spec:
     # see https://bookkeeper.apache.org/docs/latest/admin/autorecovery/
     autoRecovery: true
 
+    # To enable bookkeeper metrics feature
+    # See http://bookkeeper.apache.org/docs/4.7.0/admin/metrics/ for more metrics provider
+    options:
+      enableStatistics: "true"
+      statsProviderClass: "org.apache.bookkeeper.stats.codahale.CodahaleMetricsProvider"
```

Comment:
Is this configuration sufficient? Don't we need to configure endpoints?

http://bookkeeper.apache.org/docs/4.7.0/reference/config/#statistics

Reply:
Right, for this to work, user would need to configure the endpoint. @Tristan1900 can you add a sample endpoint configuration?

Reply:
Yes, please call out the connection details (protocol, host and port) for CodahaleMetricsProvider and the default values.

Reply:
Also please call out the output frequency. From operation perspective it may be necessary to have consistent output frequency across Bookie and Pravega, to easy the querying and reading of metrics.

Reply:
Thanks for the comments guys. I am adding more details to this.